js/siteorigin-panels/dialog/widget.js

```diff
@@ -156,12 +156,12 @@ module.exports = panels.view.dialog.extend( {
         if( !this.model.get('missing') ) {
             // Only get the values for non missing widgets.
             var values = this.getFormValues();
-            if (typeof values.widgets === 'undefined') {
+            if ( typeof values.widgets === 'undefined' ) {
```

Comment:
Ah right, the WordPress JS conventions are largely derived from the jQuery style guide. Cool, I'll create an issue for this in the PB repo.